runtime/doc/luarefvim.txt

```diff
@@ -0,0 +1,5616 @@
+
+  #o#L. H. de Figueiredo, R. Ierusalimschy, and W. Celes.,
+     "Lua: an extensible embedded language".
+    @Dr. Dobb's Journal@#21##12 (Dec 1996) 26-33.
+
+  #o#R. Ierusalimschy, L. H. de Figueiredo, and W. Celes.,
+     "The evolution of an extension language: a history of Lua".
+    @Proc. of V Brazilian Symposium on Programming Languages@(2001) B-14-B-28.
+
+
+==============================================================================
+B  COPYRIGHT & LICENSES                                        *luaref-copyright*
+==============================================================================
+
+
+This help file has the same copyright and license as Lua 5.1 and the Lua 5.1
+ manual:
+
+Copyright (c) 1994-2006 Lua.org, PUC-Rio.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+
+==============================================================================
+C  LUAREFVIM DOC               *luarefvim* *luarefvimdoc* *luaref-help* *luaref-doc*
+==============================================================================
+
+
+This is a Vim help file containing a reference for Lua 5.1, and it is -- with
+a few exceptions and adaptations -- a copy of the Lua 5.1 Reference Manual
+(see |luaref-bibliography|). For usage information, refer to |luaref-docUsage|; for
+instalation, refer to |luaref-docInstall|.
+
+This manual is composed of three parts: this file,$luarefvim.txt$, that is,
+the manual itself; a plugin file, $luarefvim.vim$, defining key bindings; and
+a help file syntax extension, $help.vim$. See |luaref-docInstall| for more
+information on these files. For copyright information, see |luaref-copyright|.
+
+The main ideas and concepts on how to implement this reference were taken from
+Christian Habermann's CRefVim project
+($http://www.vim.org/scripts/script.php?script_id=614$).
+
+
+==============================================================================
+C.1  Installation                                             *luaref-docInstall*
```

Comment:
entire C.1 and C.2 are irrelevant when shipped in runtime, could just be deleted